getting_started.md

```diff
@@ -0,0 +1,151 @@
+```
+
+In order to keep communications between devices and Overwatch secure, each device is identified through an ssh key created at install time which will be shown on your device after running the bootstrap script; you can then either register the device in the Overwatch web wizard by copy/pasting the ssh key into the UI, or you can use the agent itself for device registration with it's [command line interface](#agent-command-line-usage).
+
+Once the registration is complete your device will be able to connect to our servers, start sending health reports and download and enforce any security policies you set up for it.
+
+### Agent command line usage
+
+The Overwatch agent has a command line interface that allows you to perform basic administrative tasks, you should have the executable available as `overwatchd` in your `PATH`.
+
+#### Register agent through CLI.
+
+The agent can register itself without the need to access the Overwatch web.
+
+##### Register interactively
+
+```bash
+$ sudo overwatchd --register --name "Media Center Raspberry Pi"
+```
+
+This will prompt an interactive email/password input and register the device under the ownership of the authenticating user, the `--name` flag is optional, Overwatch will generate a name if none is specified.
+
+##### Register non-interactively
+
+Alternatively, you can specify a `--token <token>` to skip the interactive prompt, this token is associated to your account and will also register the device under your user, this is specially useful when installing the agent in bulk on multiple devices.
+
+```bash
+$ sudo overwatchd --register --name "Smoke detector" --token Xdj2kdlajk3dfjsk2j31hdlrgk3od12isdjiqk2m22i3jdfui
+```
+
+##### Get registration token
+
+You can get your registration token at any time with the `--get-token` flag, it will ask for login credentials interactively.
+
+```bash
+$ overwatchd --get-token
+```
+
+##### Regenerating identity file
+
+You can recreate the ssh identity keys at any point, if you do you will need to register the device again, as it will appear to Overwatch as a completely new and unrelated device.
+
+```bash
+$ sudo overwatchd --generate-keys --override
+```
+
+#### Starting the agent manually
+
+The Overwatch agent is installed as a service on your device, as such, you'll be able to issue standard service commands to control it's behavior.
+
+##### On Upstart systems.
+```bash
+$ sudo service overwatchd start
+$ sudo service overwatchd restart
+$ sudo service overwatchd stop
+```
+
+#### Supported Devices
+
+Overwatch bootstrap script currently supports:
+
+* upstart-based Ubuntu Linux (by default, Ubuntu 14.10 and older)
+
+We have plans to support additional IoT focused platofrms and hardware and in the near future. In the meantime, if you'd like to run Overwatch in a platform that is not listed here [let us know](get-in-touch) and we'll look into adding it to our bootstrap process.
+
+## The Overwatch Web Interface
```

Comment:
This whole section should probably be moved to another file. Simpler doc pages that deal with a single thing are better than having to scan through a very long file to find something.